### DISCUSSION TITLE
Alterações Informe Técnico 2024.002

### DISCUSSION THREAD

**Discussion by Markosbn** [wmixvideo/nfe#979] [Geral] [answered]
Bom dia, gostaria de saber se as alterações referente o informe tecnico citado estão já sendo implementadas por alguem? Se trata da atualização da tabela de formas de pagamento da nfe, onde ha um novo pix devido a nova validação feita para a opção 17 que agora é pix dinamico. 

https://www.nfe.fazenda.gov.br/portal/exibirArquivo.aspx?conteudo=T1W3L10vCC4=

**Comment by marcoslombardi** [comment, marked as answer]
Fiz um Pull Requests 980 com parte dessa NT 2024.002

**Comment by Markosbn** [comment]
@marcoslombardi vi seu commit, vou fazer alguns testes com a versão. Obrigado.
